client/package.json

```diff
@@ -119,6 +119,7 @@
     "po2json": "^1.0.0-beta-2",
     "prettier": "^2.7.1",
     "ts-node": "^10.7.0",
+    "tsconfig-paths": "^4.1.0",
```

Comment:
I'm not sure if this should be included in this PR

Reply:
Probably not, I'll remove it

Reply:
`client/package-lock.json` needs to be reset too then.